### PR TITLE
P19_Rectangle_Perimeter_Test

### PR DESCRIPTION
**Pull request by DeedeeDL**
This will make a unit testing for a progra where it is calculater the perimeter of the rectangle

### CODE REVIEW

File: src/main/java/org/example/P19_rectanglePerimeter.java [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
 public class P19_rectanglePerimeter {
-    public static void main(String[] args) {
-        Scanner scanner = new Scanner(System.in);
-        System.out.println("Enter the width of the rectangle: ");
-        double width = scanner.nextDouble();
-        System.out.println("Enter the height of the rectangle: ");
-        double height = scanner.nextDouble();
-
-        System.out.println("The perimeter of the rectangle is : " + 2 * (width + height));
+    public static int sum(int width, int height) {
```
Comment (andreiursudev) [inline src/main/java/org/example/P19_rectanglePerimeter.java]: call this method `perimeter`

File: src/test/java/org/example/P19_rectanglePerimeterTest.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class P19_rectanglePerimeterTest {
+    @Test
+    public void perimeter() {
+
+        int width = 5;
+        int height = 6;
+        int sum = P19_rectanglePerimeter.sum(width, height);
```
Comment (andreiursudev) [inline src/test/java/org/example/P19_rectanglePerimeterTest.java]: this variable should be called `perimeter` not `sum`

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/test/java/org/example/P19_rectanglePerimeterTest.java (hunk 2 above)
Comment (DeedeeDL) [reply src/test/java/org/example/P19_rectanglePerimeterTest.java]: done

Comment (andreiursudev) [reply src/test/java/org/example/P19_rectanglePerimeterTest.java]: The variable is still called `sum`

Comment (DeedeeDL) [reply src/test/java/org/example/P19_rectanglePerimeterTest.java]: modified

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**